spec/Result/OkSpec.php

```diff
@@ -42,7 +42,8 @@ function it_maps()
     function it_doesnt_errMap()
     {
         $this->beConstructedWith("foo");
-        $result = $this->mapErr(function($value) {});
+        $result = $this->mapErr(function ($value) {
+        });
```

Comment:
Expanding empty closures from `{}` to `{\n}` is a bit ugly but not a hill I would die on ^^